backend/src/main/java/cz/syntaxbro/erpsystem/configs/SecurityConfig.java

```diff
@@ -0,0 +1,43 @@
+package cz.syntaxbro.erpsystem.configs;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
+@EnableWebSecurity
+public class SecurityConfig {
+
+    @Bean
+    public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
+        http
+                .csrf(csrf -> csrf.disable()) // Disable CSRF (use with caution in production)
```

Comment:
Could be replaced by method reference
```suggestion
                .csrf(AbstractHttpConfigurer::disable)
```

Reply:
We could use dot env right from the start of project